docs/data-management/metrics/ratio-metric.md

```diff
@@ -3,15 +3,36 @@ sidebar_position: 2
 ---
 
 # Ratio metrics
+## Overview
+Ratio metrics allow you to calculate the ratio of two fact aggregations. Unlike [Simple metrics](/data-management/metrics/simple-metric), which normalize a fact aggregation by the number of unique [entities](https://docs.geteppo.com/data-management/entities) (e.g., users) in the experiment, 
+ratio metrics normalize a fact aggregation by another user-defined fact aggregation. To make this concrete, let's consider a few examples.
 
-Ratio metrics allow you to calculate ratio of two [simple metrics](/data-management/metrics/simple-metric), providing deeper insights into the relative performance of variations. This enables a more nuanced analysis of experimental results, allowing businesses to understand the impact of changes in a more comprehensive manner.
-
-For example, consider an _average order value_ metric, which is created by dividing revenue (sum of prices) by number of orders (_sum_ of items purchased or _count_ of prices).
+### Example 1: Average Order Value (AOV)
+Average order value is defined as the total amount of revenue divided by the total number of orders:
 
 $$
 \text{Average order value} = \frac{\text{Revenue}}{\text{Number of orders}}
 $$
 
+This metric is often calculated in experiments that are randomized at the user level. However, it is clear from this metric's definition that we need to normalize by the
+number of **orders** rather than the number of **users** (which simple metrics would do automatically). To capture this custom aggregation logic in the denominator, we define it as a ratio metric.
+The numerator aggregation is the sum of revenue, and the denominator aggregation is the count of orders.
+
+### Example 2: Average revenue per paying user (ARPPU)
```

Comment:
@emetelka I know you mentioned that you've typically seen this acronymized as ARPU, but I think having the extra "P" here for "paying" makes it clearer for folks like me who come from an industry where ARPU stands for "average revenue per user" and it includes all assigned users. Happy to change it if you disagree, though